server/lib/nl/config_builder/bar.py

```diff
@@ -85,9 +88,22 @@ def multiple_place_bar_block(column,
 
   tile.bar_tile_spec.max_variables = _MAX_VARIABLE_LIMIT
   tile.bar_tile_spec.max_places = _MAX_PLACES_LIMIT
-  # Always show top ones by default since we truncate #vars.
-  tile.bar_tile_spec.sort = BarTileSpec.DESCENDING
-  if RankingType.LOW in ranking_types and RankingType.HIGH not in ranking_types:
-    tile.bar_tile_spec.sort = BarTileSpec.ASCENDING
+  if sort_order:
+    tile.bar_tile_spec.sort = sort_order
   column.tiles.append(tile)
   return stat_var_spec_map
+
+
+def get_sort_order(state: PopulateState, cspec: ChartSpec):
+  # Use no default sort_order for special DC, since UN
+  # want the order to be as provided in variable groupings.
+  sort_order = None if is_special_dc(state.uttr.insight_ctx) \
+    else BarTileSpec.DESCENDING
+
+  if (RankingType.LOW in cspec.ranking_types and
+      RankingType.HIGH not in cspec.ranking_types):
+    sort_order = BarTileSpec.ASCENDING
+  elif RankingType.HIGH in cspec.ranking_types:
+    sort_order = BarTileSpec.DESCENDING
```

Comment:
this logic is slightly different from before for the case of both RankingType.HIGH and RankingType.LOW showing up in ranking_types or if neither show up, is that something you meant to change? Because before, those cases would still sort in descending order & now it won't do any sorting.